Maquetacion/scss/equalizer.scss

```diff
@@ -1,9 +1,10 @@
 $equalizer-black: #000;
 $equalizer-grey: #999;
+$font-page: 'Roboto', sans-serif;
 
 * {
   box-sizing: border-box;
-  font-family: sans-serif;
+  font-family: $font-page;
```

Comment:
Try not to edit the equalizer file because this may lead to unexpected results. I think it'd be better if you move the `page-font` variable to the `variables` file or another file like `fonts.scss`. 🙂

Reply:
👍